File: WebShopAppBackend/webshop/src/main/java/org/vs/dto/PrijavaOdgovor.java
package org.vs.dto;

import lombok.*;
import jakarta.validation.constraints.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class PrijavaOdgovor {

	@NotBlank
	@Pattern(regexp = "^[a-zA-Z0-9_]*$")
	@Size(min = 3, max = 100)
	private String korisnickoIme;

	private String avatarUrl;
	
	@NotBlank
	private String jwtToken;
	
	@NotBlank
	private String refreshJwtToken;

}

File: WebShopAppFrontend/web-shop-app/src/app/model/prijavaOdgovor.ts
export class PrijavaOdgovor {

    korisnickoIme: string;
    avatarUrl: string;
    jwtToken: string;
    refreshJwtToken: string;

    constructor(korisnickoIme: string,
        avatarUrl: string,
        jwtToken: string,
        refreshJwtToken: string) {
        this.korisnickoIme = korisnickoIme;
        this.avatarUrl = avatarUrl;
        this.jwtToken = jwtToken;
        this.refreshJwtToken = refreshJwtToken;
    }

}
